tests/TestDeferredTestReporter.cpp

```diff
@@ -54,7 +54,7 @@ namespace UnitTest
       {
          reporter.ReportTestStart(details);
 
-         DeferredTestResult const& result = reporter.GetResults().at(0);
+         DeferredTestResult const& result = reporter.GetResults()[0];
```

Comment:
The STL version in VxWorks doesn't implement the .at() method.  That's why I had to modify the this test file.